test/framework/universal_app.go

```diff
@@ -220,11 +226,7 @@ func NewUniversalApp(t testing.TestingT, clusterName, dpName, mesh string, mode
 	app.allocatePublicPortsFor("22")
 
 	if mode == AppModeCP {
-		app.allocatePublicPortsFor("5678", "5680", "5681", "5682", "5685", "9901")
-	}
-
-	if dpName == AppEgress {
-		app.allocatePublicPortsFor("9901")
```

Comment:
I think this was useful when debugging right? You can't allocate new ports for an already running container. I know you can always exec and curl but it's a bit easier when it's exposed 🤷

Reply:
https://github.com/kumahq/kuma/pull/13033#discussion_r2005635536